screen/transactions/src/main/java/com/ivy/transactions/TransactionsScreen.kt

```diff
@@ -252,14 +249,176 @@ private fun BoxWithConstraintsScope.UI(
     onPayOrGet: (Transaction) -> Unit = {},
     onSkipTransaction: (Transaction) -> Unit = {},
     onSkipAllTransactions: (List<Transaction>) -> Unit = {},
-    onChoosePeriodModal: (ChoosePeriodModalData?) -> Unit
+    onChoosePeriodModal: (ChoosePeriodModalData?) -> Unit,
 ) {
-    val ivyContext = ivyWalletCtx()
-    val itemColor = (account?.color ?: category?.color?.value)?.toComposeColor() ?: Gray
-
     var categoryModalData: CategoryModalData? by remember { mutableStateOf(null) }
     var accountModalData: AccountModalData? by remember { mutableStateOf(null) }
 
+    HeaderUI(
```

Comment:
Hey @muray52 I'm afraid of these changes because they're hard to review and verify. Can you revert them and `Suppress` Detekt errors that are hard to fix. Please read our guide:
https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md

Reply:
Yeah, I know.
I reverted commit and add Suppress()

Reply:
You can use Suppress() for Lint errors, as well
Also delete the not used state from the VM